mergify_engine/actions/queue.py

```diff
@@ -42,6 +42,13 @@
 
 
 class QueueAction(merge_base.MergeBaseAction):
+    UNQUEUE_DOCUMENTATION = f"""
+You can take a look to `{constants.MERGE_QUEUE_SUMMARY_NAME}` check-runs for more details.
+
+In case of a failure due to a flaky test, once you have retriggered the CI, you
+can re-embarked the pull request for merge with the comment `@mergifyio refresh`.
+"""
```

Comment:
```suggestion
You can take a look at `{constants.MERGE_QUEUE_SUMMARY_NAME}` check runs for more details.

In case of a failure due to a flaky test, you should first retrigger the CI. Then, re-embark the pull request into the merge queue by posting the comment `@mergifyio refresh` on the pull request.
"""
```